src/domain/entities/Questionnaire/QuestionnaireQuestion.ts

```diff
@@ -24,6 +25,35 @@ export interface QuestionBase {
     stageId?: Id; //For repeatable stages processing.
 }
 
+export interface SpeciesQuestion extends SelectQuestion {
+    subType: "select-species";
+}
+
+export interface AntibioticQuestion extends SelectQuestion {
+    subType: "select-antibiotic";
+    filteredOptions?: QuestionOption[];
+}
+
+export const isSpeciesQuestion = (question: SelectQuestion): question is SpeciesQuestion => {
+    return (question as SpeciesQuestion).subType === "select-species";
+};
+
+export const isAntibioticQuestion = (question: SelectQuestion): question is AntibioticQuestion => {
+    return (question as AntibioticQuestion).subType === "select-antibiotic";
+};
+
+const ANTIBIOTIC_QUESTION_FORM_NAME = "Specify the antibiotic";
```

Comment:
Could this be use in questionHelper.ts? Instead of:
```typescript
const isAntibioticQuestion = name.startsWith(`Specify the antibiotic`);
``` 
something like
```typescript
const isAntibioticQuestion = name.startsWith(ANTIBIOTIC_QUESTION_FORM_NAME);
``` 

And also for species, for having the name centralized in some point